extension/src/plots/model/collect.ts

```diff
@@ -376,6 +477,80 @@ export const collectSelectedComparisonPlots = ({
   return acc
 }
 
+const collectSelectedImgComparisonPlotClasses = ({
+  comparisonClassesSelected,
+  acc,
+  img,
+  id,
+  path
+}: {
+  comparisonClassesSelected: ComparisonClassesSelected
+  acc: ComparisonPlotClasses
+  img: ImagePlot
+  id: string
+  path: string
+}) => {
+  const imgClasses: ComparisonPlotClass[] = []
+
+  for (const [label, boxes] of Object.entries(img.boundingBoxes || {})) {
+    const selectedState = comparisonClassesSelected[path]?.[label]
+
+    if (selectedState === false) {
+      continue
+    }
+
+    imgClasses.push({
+      boxes: boxes.map(({ x_min, x_max, y_min, y_max }) => ({
+        h: y_max - y_min,
+        w: x_max - x_min,
+        x: x_min,
+        y: y_min
```

Comment:
We could also perform this calculation (getting `h`, `w`, `x`, `y`) when we're collecting `comparisonData` from the DVC output.

Reply:
We now perform this calculation inside of just the react img file.